build/build.sh

```diff
@@ -13,26 +29,29 @@ then
 
 	if [[ $TRAVIS_BRANCH == "master" && $(cat version.sbt) != *"SNAPSHOT"* ]]
 	then
-		eval "$(ssh-agent -s)"
-		chmod 600 local.deploy_key.pem
-		ssh-add local.deploy_key.pem
-		git config --global user.name "Quill CI"
-		git config --global user.email "quillci@getquill.io"
-		git remote set-url origin git@github.com:getquill/quill.git
-		git fetch --unshallow
-		git checkout master || git checkout -b master
-		git reset --hard origin/master
-		git push --delete origin website
-		sbt tut 'release cross with-defaults'
+	    if [[ $SCALA_VERSION == '2.11' ]]
```

Comment:
The indentation seems off here. Also, could you add a comment explaining why this condition is necessary?